RELEASE/scripts/autoscend/auto_heavyrains.ash

```diff
@@ -643,9 +645,8 @@ boolean L13_towerFinalHeavyRains()
 	}
 	else foreach it in $items[Rain-Doh green lantern, meteorb, snow mobile]
 	{
-		if(creatable_amount(it) > 0)
+		if(haveMakeOrPullOne(it))
```

Comment:
perhaps rename `acquireWithOnePull`?

Reply:
change made (to a different name we discussed in discord)